trunk/SUAVE/Methods/Flight_Dynamics/Dynamic_Stability/compute_aero_derivatives.py

```diff
@@ -0,0 +1,134 @@
+## @ingroup Methods-Flight_Dynamics-Dynamic_Stability
+# compute_aero_derivatives.py
+# 
+# Created:   Aug 2021, R. Erhard
+# Modified: 
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+import SUAVE
+from SUAVE.Core import Data
+
+import numpy as np 
+from copy import deepcopy
+
+## @ingroup Methods-Flight_Dynamics-Dynamic_Stability
+def compute_aero_derivatives(segment, h=1e-4): 
+    """This function takes an aircraft with analyses setup and initial state,
+    and computes the aerodynamic derivatives about the initial state.
+    
+    Assumptions:
+       Linerarized equations are used for each state variable
+
+    Source:
+      N/A
+
+    Inputs:
+      vehicle                SUAVE Vehicle 
+      state                  Initial state of vehicle (after running mission segment of interest)
+
+    Outputs: 
+       aero_derivatives      Aerodynamic derivatives wrt perturbed state variables
+
+    Properties Used:
+       N/A
+     """
+    
+    # extract states from converged mission segment
+    velocity_vector    = segment.state.conditions.frames.inertial.velocity_vector
+    orientation_vector = segment.state.conditions.frames.body.inertial_rotations
+    
+    alpha           = orientation_vector[:,1]
```

Comment:
This is not technically alpha. It's pitch angle

Reply:
fixed